queries/pkl/highlights.scm

```diff
@@ -132,21 +128,20 @@
 "amends" @keyword
 "as" @keyword
 "class" @keyword
-"else" @conditional
+"else" @keyword.conditional ; TODO: fix 'else' not highlighting after 'when'
 "extends" @keyword
 "external" @keyword
 (falseLiteral) @boolean
-"for" @repeat
-"function" @keyword
+"for" @keyword.repeat
+"function" @keyword.function
 "hidden" @keyword
-"if" @conditional
-(importExpr "import" @include)
-(importGlobExpr "import*" @include)
-(importClause "import" @include)
-(importGlobClause "import*" @include)
-(importClause "as" @include)
-"in" @repeat
-"is" @keyword.operator
+"if" @keyword.conditional
+(importExpr "import" @module)
+(importGlobExpr "import*" @module)
+"import" @keyword.import
+"import*" @keyword.import
```

Comment:
Neovim used last-win order (and so is Helix and upstream Treesitter as of now). This made it so only keyword.import is shown

Reply:
Can you please clarify this? Should I correct it to just `@keyword` instead of '@keyword.import' as it was before?

Reply:
If two captures to the same node, i.e

```
(Foo (a) @capture1)
(a) @capture2  
```

Then the capture appear last will be used for highlighting

Tl;dr: cascading highlights